sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ChangeFeedProcessorState.java

```diff
@@ -98,8 +98,8 @@ public int getEstimatedLag() {
      * @param estimatedLag the estimated lag.
      * @return the current ChangeFeedProcessorState instance.
      */
-    public ChangeFeedProcessorState setEstimatedLag(int estimatedLag) {
-        this.estimatedLag = estimatedLag;
+    public ChangeFeedProcessorState setEstimatedLag(long estimatedLag) {
```

Comment:
This is a breaking change, isn't it?

Reply:
yea agree - discussed with Kushagra earlier and agree that on public API level we should keep the same, and no need for breaking change. I will converted the change back in the next iteration iteration

Reply:
updated